hail/src/main/scala/is/hail/types/physical/stypes/concrete/SUnreachable.scala

```diff
@@ -251,17 +251,19 @@ class SUnreachableNDArrayValue(val st: SUnreachableNDArray) extends SUnreachable
 
   def loadElementAddress(indices: IndexedSeq[is.hail.asm4s.Value[Long]],cb: is.hail.expr.ir.EmitCodeBuilder): is.hail.asm4s.Code[Long] = const(0L)
 
-  def shapes(cb: EmitCodeBuilder): IndexedSeq[Value[Long]] = (0 until st.nDims).map(_ => const(0L))
+  def shapes: IndexedSeq[SizeValue] = (0 until st.nDims).map(_ => SizeValueStatic(0L))
 
-  def strides(cb: EmitCodeBuilder): IndexedSeq[Value[Long]] = (0 until st.nDims).map(_ => const(0L))
+  def strides: IndexedSeq[Value[Long]] = (0 until st.nDims).map(_ => const(0L))
```

Comment:
I'm reconsidering whether removing the CodeBuilders is a good thing to do. I think it made something easier, but I can't remember anymore. I'm going to try to revert that change.

Reply:
Actually, revisiting this I think I prefer leaving the CodeBuilders off. The only benefit to putting them back is to make it easier to change the representation of SNDArrays to store strides in #elements, but I don't think that would be a difficult refactoring regardless. And leaving them off makes it easier to use strides without worrying about storing in locals.

Happy to put them back if others prefer.